coc.md

```diff
@@ -32,7 +32,6 @@ layout: page
 					<li>Threats of violence</li>
 					<li>Incitement of violence towards any individual, including encouraging a person to commit suicide or to engage in self-harm</li>
 				</ul>
-			<p>Jokes that resemble such offensive comments are still considered offensive even if meant ‘ironically’.</p>
```

Comment:
The original placement seemed odd, since it was right in the middle of a
list as its own paragraph almost like a non-sequitor. Maybe a different
list item? Or bolding?

*Creatrix Tiara*
Interdisciplinary Iconoclast and Creatrix of Awesome
http://creatrixtiara.com

Interested in feminist & queer access to sexual health? Join us:
http://screetreboot.github.io
Help me get to the School for Poetic Computation!
https://www.tilt.com/tilts/help-tiara-attend-school-for-poetic-computation/

On 15 August 2015 at 20:59, Aria Stewart <notifications@github.com> wrote:

> In coc.md
> <https://github.com/lgbtq-technology/lgbtq-technology.github.io/pull/11#discussion_r37141704>
> :
>
> > @@ -32,7 +32,6 @@ layout: page
> >  					<li>Threats of violence</li>
> >  					<li>Incitement of violence towards any individual, including encouraging a person to commit suicide or to engage in self-harm</li>
> >  				</ul>
> > -			<p>Jokes that resemble such offensive comments are still considered offensive even if meant ‘ironically’.</p>
>
> Hmm. I'd actually deliberately moved this down -- I do want to call racism
> out specifically, but it might actually apply generally. Thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lgbtq-technology/lgbtq-technology.github.io/pull/11/files#r37141704>
> .
>